packages/unuse-angular/src/index.spec.ts

```
@@ -0,0 +1,190 @@
+  overrideTryOnScopeDisposeFn,
+  overrideUnResolveFn,
+  toUnSignal,
+  tryOnScopeDispose,
+  unComputed,
+  unResolve,
+  unSignal,
+} from '.';
+
+@NgModule({ providers: [provideZonelessChangeDetection()] })
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export class ZonelessTestModule {}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)